site/src/comparison.rs

```diff
@@ -544,6 +562,15 @@ pub fn write_summary_table(
             .unwrap_or_else(|| "-".to_string())
     }
 
+    fn render_range<F: FnOnce() -> (f64, f64)>(count: usize, calculate: F) -> String {
+        if count > 0 {
+            let (a, b) = calculate();
+            format!("[{a:.1}%, {b:.1}%]")
```

Comment:
I agree that using the same range notation as the comparison page is the right move. I do wonder if the `[a, b]` notation is right or using a Rust's range notation (`..=`) would be clearer.

Reply:
I think `a..=b` is one of Rust's least nice pieces of syntax, and that `[a, b]` is better.